assets/src/edit-story/elements/text/edit.js

```diff
@@ -199,51 +188,11 @@ function TextEdit({
     evt.stopPropagation();
   };
 
-  // Finally update content for element on unmount.
-  useEffect(
-    () => () => {
-      const newState = lastKnownState.current;
-      const newHeight = editorHeightRef.current;
-      wrapperRef.current.style.height = '';
-      if (newState) {
-        // Remember to trim any trailing non-breaking space.
-        const properties = {
-          content: stateToHTML(lastKnownState.current, {
-            defaultBlockTag: null,
-          }).replace(/&nbsp;$/, ''),
-        };
-        // Recalculate the new height and offset.
-        if (newHeight) {
-          const [dx, dy] = calcRotatedResizeOffset(
-            rotationAngle,
-            0,
-            0,
-            0,
-            newHeight - height
-          );
-          properties.height = editorToDataY(newHeight);
-          properties.x = editorToDataX(x + dx);
-          properties.y = editorToDataY(y + dy);
-        }
-        setProperties(properties);
-      }
-    },
-    [setProperties, x, y, height, rotationAngle, editorToDataY, editorToDataX]
-  );
```

Comment:
Find it unnecessary, since we have update elements with new properties on useDesignPanel. @dvoytenko I need confirm for this one.

Reply:
This update is completely unrelated to the design panel. It updates when the text content itself updates.

Reply:
Yes, it should be. Added new update when the element height updated through outside design panel